src/api/java/jeresources/api/drop/LootDrop.java

```diff
@@ -186,12 +186,21 @@ private String getDropChance() {
         return chance < 1F ? " (" + formatChance() + "%)" : "";
     }
 
-    private String formatChance() {
+    public String formatChance() {
         float chance = this.chance * 100;
         if (chance < 10) return String.format("%.1f", chance);
         return String.format("%2d", (int) chance);
     }
 
+    /**
+     * Use by world gen only
+     * Check if conditionals contains Conditional.affectedByFortune
+     * @return True if Conditional.affectedByFortune is found, False if it not
+     */
+    public boolean isAffectedByFortune() {
```

Comment:
Can we also add a general function for this, seems like a useful tool to me.
`public boolean isAffectedBy(Conditional)`